packages/data-stores/src/reader/helpers/index.ts

```diff
@@ -89,17 +90,22 @@ const getSubscriptionMutationParams = (
 	isLoggedIn: boolean,
 	blogId?: number | string,
 	url?: string,
-	emailId?: string
+	emailId?: string,
+	subscriptionId?: number
```

Comment:
Consider adding a check to ensure that the `subscriptionId` is valid before proceeding with the subscription process. This will help prevent potential issues if an invalid `subscriptionId` is passed. [important]